models/agent.ts

```diff
@@ -0,0 +1,23 @@
+import dbClient from "../lib/db";
+
+export async function createAgent(agent) {
+  const id = await dbClient.agent.create({data: agent, select: id});
+  return id;
+}
+
+export async function findByAgent(agent) {
+    const agentId = await dbClient.agent.findFirst({
+     where: {
+       mbox: agent.mbox? agent.mbox : null,
+       mbox_sha1sum: agent.mbox_sha1sum? agent.mbox_sha1sum : null,
+       openid: agent.openId? agent.openid : null
```

Comment:
Shouldn't we have a comma (`,`) at the end of the line? Strangely, the test suite is not breaking... hmm...